Dockerfile

```
@@ -1,8 +1,9 @@
 # Meeting Bot - Docker Image for Screen Recording
 FROM ubuntu:24.04
 
+RUN apt-get update && apt-get install -y curl ca-certificates gnupg wget
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)